spec/features/hiring_staff_can_copy_a_vacancy_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+RSpec.feature 'Copying a vacancy' do
+  let(:school) { create(:school) }
+
+  before(:each) do
+    stub_hiring_staff_auth(urn: school.urn)
+  end
+
+  scenario 'hiring staff can see a Duplicate link on published and pending jobs' do
+    create(:vacancy, school: school, status: 'published')
+    create(:vacancy,
+           school: school,
+           status: 'published',
+           expires_on: Time.zone.today + 4.days,
+           publish_on: Time.zone.tomorrow)
+
+    visit school_path
+
+    published_table = page.find('section#published-jobs table')
```

Comment:
These tests are specific about where in the DOM they expect to find the content. I tend to find being specific in 1 test makes it much easier to accommodate changes to the DOM in future.

In this case we now know we're going to have to change this layout imminently from the design system to something with links, and there's a good chance that 4 of these tests will have to be reworked to get things to go green. 

If you extrapolate that approach out across the whole application, changing something in a common view (root_path) could take someone a very long time to work through and could be quite painful for them.

My advice would be to have 1 test for the happy path of this feature that _does_ go into specifics and trust that it will fail if something is unexpectedly changed. The edge cases can likely get by by asserting content is present on the page.

Reply:
These selectors are already used in another feature test: https://github.com/dxw/teacher-vacancy-service/blob/develop/spec/features/hiring_staff_can_see_their_vacancies_spec.rb

If they should be removed in this PR then the other feature test will probably need refactoring as well.